grpcurl.go

```diff
@@ -542,6 +542,14 @@ func ClientTransportCredentials(insecureSkipVerify bool, cacertFile, clientCertF
 		tlsConf.RootCAs = certPool
 	}
 
+	if os.Getenv("SSLKEYLOGFILE") != "" {
```

Comment:
I don't think this belongs in this helper function -- this seems too subtle for a library function, especially with no way to programmatically ignore the env var. And we can't add this flag as a new parameter since that would not be a backwards-compatible change.

Instead, I think we should add a new exported function `ClientTLSConfig` that is most of this function, but just returns `&tlsConf`. This existing function can be re-implemented to delegate to the new one. We could then move most of the Go doc to the new function and have this function's doc that it is a convenience shortcut that calls `ClientTLSConfig` followed by `credentials.NewTLS(result)`.

Once that refactor is done, the caller of this (`grpcurl`'s main method) can directly call `ClientTLSConfig` and there is where we should examine the environment variable and set the `KeyLogWriter` field, before calling `credentials.NewTLS`.

Does that all make sense?